api/src/main/java/io/strimzi/api/kafka/model/KafkaConnectorConfig.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright 2019, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.api.kafka.model;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import io.strimzi.crdgenerator.annotations.Description;
+import io.sundr.builder.annotations.Buildable;
+import lombok.EqualsAndHashCode;
+
+import java.io.Serializable;
+import java.util.HashMap;
+import java.util.Map;
+
+import static java.util.Collections.emptyMap;
+
+@Buildable(
+        editableEnabled = false,
+        generateBuilderPackage = false,
+        builderPackage = "io.fabric8.kubernetes.api.builder"
+)
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@EqualsAndHashCode
+public class KafkaConnectorConfig implements Serializable, UnknownPropertyPreserving {
+    private static final long serialVersionUID = 1L;
+
+    private String name;
+    private String value;
```

Comment:
if a configuration parameter is just a key-value pair, what's the real value of having just a `Map<String, String>` in the `KafkaConnectorSpec` instead of a `List<KafkaConnectorConfig>`?

Reply:
So is this one resolved by your comment in the discussion [above](https://github.com/strimzi/strimzi-kafka-operator/pull/2073#discussion_r335314540)?

Reply:
but now I don't see name-value in the #2087 anymore. It's going in the opposite direction. Or did I misunderstand the final decision and rationale?